hubspot/connection/exc.py

```diff
@@ -17,7 +17,10 @@
 
 class HubspotException(Exception):
     """The base HubSpot error."""
-    pass
+    def __init__(self, message):
+        """ Explicitly makes the message available as the 'message' attribute. """
+        self.message = message
```

Comment:
As @gnarea said, we shouldn't have done this in the first place, we are going to revert the specific commit that introduced this problem so we don't need to merge this commit.

Could you remove this commit from your PR? Sorry about the confusion.